vscode-client/README.md

```diff
@@ -13,7 +13,7 @@ at once with minimal extra configuration per language.
 To view a language server's stderr output in VSCode, select View → Output.
```

Comment:
Please remove this folder (i.e. `vscode-client`) because this support is obsolete (no one has contributed to it in years).